File: examples/bluetooth-car/code.js
bluetooth.onBluetoothConnected(function () {
    bluetooth.startLEDService()
    bluetooth.startButtonService()
})
dcmotor.usedc()
basic.forever(function () {
    if (input.buttonIsPressed(Button.AB)) {
        dcmotor.m1(true, 50)
        dcmotor.m2(true, 50)
    } else if (input.buttonIsPressed(Button.A)) {
        dcmotor.m1(false, 35)
        dcmotor.m2(true, 35)
    } else if (input.buttonIsPressed(Button.B)) {
        dcmotor.m1(true, 35)
        dcmotor.m2(false, 35)
    } else {
        dcmotor.m1(false, 0)
        dcmotor.m2(false, 0)
    }
})

File: examples/bluetooth-car/code.py
def on_bluetooth_connected():
    bluetooth.start_led_service()
    bluetooth.start_button_service()
bluetooth.on_bluetooth_connected(on_bluetooth_connected)

dcmotor.usedc()

def on_forever():
    if input.button_is_pressed(Button.AB):
        dcmotor.m1(True, 50)
        dcmotor.m2(True, 50)
    elif input.button_is_pressed(Button.A):
        dcmotor.m1(False, 35)
        dcmotor.m2(True, 35)
    elif input.button_is_pressed(Button.B):
        dcmotor.m1(True, 35)
        dcmotor.m2(False, 35)
    else:
        dcmotor.m1(False, 0)
        dcmotor.m2(False, 0)
basic.forever(on_forever)
